src/server.rs

```diff
@@ -407,6 +415,11 @@ impl Future for ShutdownFuture {
 unsafe impl Sync for Inner {}
 unsafe impl Send for Inner {}
 
+/// A gRPC server.
+///
+/// A single server can server arbitrary number of services and can listen on more than one ports.
```

Comment:
-> /// A single server can **serve**("provide" ?) arbitrary number of services and can listen on more than one **port.**
Note: Even though "more than one" indicates a plural meaning, it should go before a singular noun.